CLAUDE.md

```
@@ -226,8 +243,21 @@ public void handleCommentCreated(CommentCreatedEvent event) {
 }
 ```
 
-### 3. TSID
+### 3. Facade 패턴
+복합 도메인 로직을 단순화하는 퍼사드 계층 (letter 도메인에서 사용)
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)